packages/nextjs/app/streamer/_components/Guru.tsx

```diff
@@ -5,6 +5,7 @@ import { Address } from "~~/components/scaffold-eth";
 import { useDeployedContractInfo, useWatchBalance } from "~~/hooks/scaffold-eth";
 
 export const STREAM_ETH_VALUE = "0.5";
+export const ETH_PER_CHARACTER = "0.01";
```

Comment:
Makes more sense for this constant to be contained with _Guru.tsx_, as Guru is the one setting the price.
Also, this constant could be necessary to perform a side quest to close off a Rube who isn't paying up (see [Checkpoint 3 > Side Quest](https://speedrunethereum.com/challenge/state-channels)).